modules/oidcsso/src/main/java/org/wso2/carbon/tomcat/oidcsso/extension/Constants.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.tomcat.oidcsso.extension;
+
+/**
+ * This class defines constants used within the tomcat extension for OpenID Connect.
+ */
+public class Constants {
+    public static final String OPENID = "openid";
+    public static final String ERROR_AUTHENTICATION_RESPONSE = "Error Authentication Response";
+    public static final String AUTHENTICATION_RESPONSE_VALIDATION_FAILED = "Authentication Response Validation Failed";
+    public static final String TOKEN_RESPONSE_VALIDATION_FAILED = "Token Response Validation Failed";
+    public static final String ERROR_TOKEN_RESPONSE = "Error Token Response";
+    public static final String ERROR_USER_INFO_RESPONSE = "UserInfo Error Response";
+    public static final String USER_INFO_RESPONSE = "UserInfo response";
```

Comment:
ERROR_AUTHENTICATION_RESPONSE, AUTHENTICATION_RESPONSE_VALIDATION_FAILED, TOKEN_RESPONSE_VALIDATION_FAILED, ERROR_TOKEN_RESPONSE and ERROR_USER_INFO_RESPONSE seems unused, shall we remove them?